package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "@babel/preset-typescript": "^7.22.5",
     "@playwright/test": "^1.37.0",
+    "@replayio/playwright": "3.0.0-alpha.3",
```

Comment:
does this setup work for you with this version now?

Reply:
No. The latest comment I left in Linear mentioned that the steps are all missing. I'll add you to the workspace so you can look at them.

Reply:
Let's see what happens with the newest version ( https://github.com/bvaughn/react-resizable-panels/pull/349 ). I'm simultaneously testing this release in the overboard's repo

Reply:
Here's the latest recording: https://app.replay.io/recording/68393c17-bc53-4c80-a972-f29c9e0fa579

No steps.

Reply:
I'm able to repro this locally using ur tests here... diving in! 🤿